apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -640,6 +662,32 @@ await fake.TaskHelper.ConfigureAwaitHideCancellation(
             }
         }
 
+        [Fact]
+        public void DisposeBeforeStart()
+        {
+            using (var fake = Fake.CreateClientForSingleResponseStream(new[] { ServerAction.Inf() }))
+            {
+                fake.Scheduler.Run(async () =>
+                {
+
```

Comment:
Nit: remove empty line.

Reply:
Done, will push and merge on green. Thanks